pkg/image/apis/image/helper.go

```diff
@@ -41,6 +46,16 @@ const (
 	ImportRegistryNotAllowed = "registry is not allowed for import"
 )
 
+// knownSignatureStores is a list of known signature stores. The key represents
+// a registry name and it maps to a signature store for that registry. The
+// signature store has to be HTTPS and it must offer image signatures based on
+// this format: https://STORE_URL/IMAGE_NAME/signature-1
+// The response must be valid image signature blob.
+// TODO: Maybe this should be configurable via master config?
```

Comment:
@aweiteka image importer does not need image signer role because the controller that runs the importer has right to create images (and signature is part of the image)